dotnet-runtime-90/snap/snapcraft.yaml

```diff
@@ -0,0 +1,84 @@
+name: dotnet-runtime-90
+base: core22
+adopt-info: dotnet-runtime
+summary: .NET 9.0 Runtime.
+description: |
+  .NET Core runtimes and libraries which are optimized for running .NET Core apps in production. See https://dot.net/core.
+  .NET Core is a general purpose development platform maintained by Microsoft.
+
+grade: stable
+confinement: strict
+
+package-repositories:
+ - type: apt
+   ppa: dotnet/snaps
+   priority: always
+
```

Comment:
**note:** The other runtime `snapcraft.yaml` files have an `apps` section:

```
apps:
  dotnet:
    command: dotnet
    plugs:
      - network
      - network-bind
      - removable-media
      - home
```

**question:** Why does the `dotnet-runtime-90` snap not need it?

Reply:
Sure. Done ✅